src/core/function_messages_cqrs.tf

```diff
@@ -0,0 +1,242 @@
+locals {
+  function_app_messages_cqrs = {
+    app_settings = {
+      FUNCTIONS_WORKER_RUNTIME       = "node"
+      WEBSITE_NODE_DEFAULT_VERSION   = "14.16.0"
+      WEBSITE_RUN_FROM_PACKAGE       = "1"
+      WEBSITE_VNET_ROUTE_ALL         = "1"
+      WEBSITE_DNS_SERVER             = "168.63.129.16"
+      FUNCTIONS_WORKER_PROCESS_COUNT = 4
+      NODE_ENV                       = "production"
+
+      COSMOSDB_NAME                = "db"
+      COSMOSDB_URI                 = data.azurerm_cosmosdb_account.cosmos_api.endpoint
+      COSMOSDB_KEY                 = data.azurerm_cosmosdb_account.cosmos_api.primary_master_key
+      COSMOS_API_CONNECTION_STRING = format("AccountEndpoint=%s;AccountKey=%s;", data.azurerm_cosmosdb_account.cosmos_api.endpoint, data.azurerm_cosmosdb_account.cosmos_api.primary_master_key)
+
+      MESSAGE_CONTAINER_NAME = "message-content"
+      QueueStorageConnection = data.azurerm_storage_account.api.primary_connection_string
+
+      // Keepalive fields are all optionals
+      FETCH_KEEPALIVE_ENABLED             = "true"
+      FETCH_KEEPALIVE_SOCKET_ACTIVE_TTL   = "110000"
+      FETCH_KEEPALIVE_MAX_SOCKETS         = "40"
+      FETCH_KEEPALIVE_MAX_FREE_SOCKETS    = "10"
+      FETCH_KEEPALIVE_FREE_SOCKET_TIMEOUT = "30000"
+      FETCH_KEEPALIVE_TIMEOUT             = "60000"
+
+    }
+  }
+}
+
+resource "azurerm_resource_group" "function_messages_cqrs_rg" {
+  name     = format("%s-fn-messages-cqrs-rg", local.project)
```

Comment:
Can we use a resource group for backends messages domain or a shared cqrs?

Reply:
yes we can use a shared rg for message domain